volatility3/framework/__init__.py

```diff
@@ -22,14 +22,14 @@
         )
     )
 
-import importlib
-import inspect
-import logging
-import os
-import traceback
-from typing import Any, Dict, Generator, List, Tuple, Type, TypeVar
+import importlib  # noqa: E402
+import inspect  # noqa: E402
```

Comment:
Really don't like noqaing that many things.  We could put that into a `check_python_version` module instead, and just import it in the same place?  I don't think it affects any of the other imports, other than firing first?

Reply:
I moved them up, AFAICT these would only throw/raise if you were to use something older than python 3.1

Reply:
its now in its own module: https://github.com/volatilityfoundation/volatility3/pull/1364/commits/a2f71bf49161992bdfed0bdbd7c2d00affd83e6b